Cargo.lock

```diff
@@ -14,13 +14,14 @@ dependencies = [
 
 [[package]]
 name = "ahash"
-version = "0.7.8"
+version = "0.8.7"
```

Comment:
Why are these Cargo updates needed?

Reply:
Some of our current dependencies would break the build with this toolchain update. The fix is updating to a newer version.